MaxText/layers/linears.py

```diff
@@ -84,9 +84,7 @@ def __init__(
       axis: Union[Iterable[int], int] = -1,
       weight_dtype: DType = jnp.float32,
       dtype: DType = jnp.float32,
-      kernel_init: NdInitializer = nd_dense_init(
-          1.0, "fan_in", "truncated_normal"
-      ),
+      kernel_init: NdInitializer = nd_dense_init(1.0, "fan_in", "truncated_normal"),
```

Comment:
Do we have lint rules saying these types of parenthesis values need to be on a single line?

Reply:
Again it just seems to use `pyink` defaults as per:
> https://github.com/AI-Hypercomputer/maxtext/blob/69ffd34/code_style.sh#L31